src/frontend/src/pages/InfoPage.tsx

```diff
@@ -12,8 +12,46 @@ import ChatIcon from '@mui/icons-material/Chat';
 import ExternalLink from '../components/ExternalLink';
 import PageBlock from '../layouts/PageBlock';
 import PageLayout from '../components/PageLayout';
+import { Button } from '@mui/material';
+import { useState } from 'react';
+import DRCSummaryModal from './DesignReviewSummaryModal';
+import { DesignReview, DesignReviewStatus } from 'shared';
+import {
+  batman,
+  superman,
+  greenlantern,
+  flash,
+  aquaman,
+  wonderwoman
+} from '../../../backend/tests/test-data/users.test-data';
 
 const InfoPage: React.FC = () => {
+  const [isSummaryModalOpen, setIsSummaryModalOpen] = useState(false);
+  const handleOpenSummary = () => setIsSummaryModalOpen(true);
+  const handleCloseSummary = () => setIsSummaryModalOpen(false);
+
+  const exampleDesignReview: DesignReview = {
```

Comment:
this can go in the design-review.utils.ts file and you should add a TODO saying to delete it when we actually hook up this feature